src/Bicep.Cli/Arguments/BuildArguments.cs

```diff
@@ -48,6 +48,19 @@ public BuildArguments(string[] args) : base(Constants.Command.Build)
                         i++;
                         break;
 
+                    case "--diagnosticsformat":
```

Comment:
>diagnosticsformat [](http://example.com/codeflow?start=28&length=17)

Can we do `--diagnostic-format` instead? I think it's more consistent with the linux naming convention. (I also think that we made a mistake on --outdir and --outfile because, but it's too late to fix now.)

Reply:
changed it to `--diagnostics-format`. Is there any preference on naming it something else? I went with diagnostics but I'm not convinced it's the best option.

Reply:
Sounds good to me. (I don't have any better ideas.)